deb/ch3/ep1/answers/passengers.py

```diff
@@ -0,0 +1,75 @@
+from glob import glob
+from deb.utils.config import config
+from deb.utils.logging import logger
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import initcap, col, sha2, concat_ws
+from google.cloud import bigquery
+
+
+# ===============================================================
+# ‹‹=‹‹=‹‹=‹‹=‹‹=‹‹=‹‹=‹‹= BigQuery Util =››=››=››=››=››=››=››=››
+# ===============================================================
+
+def gbq_load(table_name, parquet_dir):
+    logger.info(f"loading bigquery table: `{table_name}` from {parquet_dir}")
+    # Construct a BigQuery client object.
+    client = bigquery.Client()
+    # Construct a BigQuery job config object.
+    job_config = bigquery.LoadJobConfig(
+        source_format=bigquery.SourceFormat.PARQUET,
+    )
+
+    data_file = glob(parquet_dir+'/part*.parquet')[0]
+    logger.info(f"loading data from {data_file}")
+    with open(data_file, "rb") as source_file:
+        job = client.load_table_from_file(source_file, table_name, job_config=job_config)
+    job.result()  # Waits for the job to complete.
+    table = client.get_table(table_name)  # get loaded table info
+    logger.info(f"loaded {table.num_rows} rows to {table_name}")
```

Comment:
You'll notice I stole the function, not the full object as is my inclination. If you want, I can wrap this in a class but I'm going to keep it mono-purpose for clarity.